src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -320,6 +320,10 @@ private void initKeyBindings() {
                         getCurrentLibraryTab().getMainTable().requestFocus();
                         event.consume();
                         break;
+                    case FOCUS_GROUP_LIST:
+                        this.stateManager.getGroupTree().requestFocus();
```

Comment:
Why a call to the state manager? Jabref frame is the topmost ui Element. It should be no problem to navigate through the children to find the grouptreeview and focus on it.